### PR TITLE
Kb test language analyzer

### PR DESCRIPTION
**Pull request by keithbradley1**
work done in test_behavior file to get averages to work
init and self.message added to the behavior_predictor file

### CODE REVIEW

File: behavior_predictor.py [hunk 1]
```diff
@@ -2,21 +2,29 @@
...
       for y in behavior_predictor_lexicon["dictator"]:
         if x == y[0]:
           dictator_counter += y[1]
-    return dictator_counter
 
+          # dictator_counter = (dictator_counter) / (matches)
+
+    return dictator_counter
 
   def check_hippie(self, message):
```
Comment (carter-harris) [inline behavior_predictor.py]: take "message" out of the argument on each of this instances

**Comment by ashwoodaa** [comment]
:+1:

**Comment by Jessicashinjo** [comment]
👍

**Comment by carter-harris** [comment]
👍

**Pull request merged**